packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx

```diff
@@ -108,7 +108,7 @@ export class Pivot extends React.Component<IPivotProps, IPivotState> {
     }
 
     return (
-      <a
+      <button
```

Comment:
Actually no, this would mean changing the props and that would be a breaking change. I think it's fine to call these "links", even if they are implemented with a `button` element. Approved. 😄